### PR TITLE
added custom styles for img tag - moved img src to header

### PR DESCRIPTION
**Pull request by nburd90**
- Edited image size and added some basic styling. 
- Image is clean and centered.
- White spacing to the right of the navbar has been removed.

### CODE REVIEW

File: about-us.php [hunk 1]
```diff
@@ -1,21 +1,24 @@
...
-<?php include('nav-bar.php'); ?>
+    <header>
+        <?php include('nav-bar.php'); ?> 
+        <img class="banner" src="/crispy-images/indiefilms.jpg">
+    </header>
 
 <div class="container">  
-<img src="crispy-images/indiefilms.jpg">
-        <h1>About Us</h1>
+
```
Comment (billythakidd04) [inline about-us.php]: delete the empty line

File: about-us.php [hunk 2]
```diff
@@ -1,21 +1,24 @@
...
-  <head>
-    
+
+<head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
+    <link rel="stylesheet" href="styles.css">
     <title>Behind the Reel</title>
   </head>
```
Comment (billythakidd04) [inline about-us.php]: delete the empty lines

File: home-page.php [hunk 3]
```diff
@@ -1,20 +1,19 @@
...
 <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
     <link rel="stylesheet" href="styles.css"/> 
     <title>Beyond the Reel</title>
-
-    <!--Will add an icon 
-    <link rel="shortcut icon" href="" type=""/> -->
 </head>
```
Comment (billythakidd04) [inline home-page.php:12]: extra empty line

**Review by billythakidd04 — APPROVED** [review]
something weird going on with the diff for news.php so make sure its correct before merging.

File: about-us.php (hunk 1 above)
Comment (nburd90) [reply about-us.php]: Deleted.

File: home-page.php (hunk 3 above)
Comment (nburd90) [reply home-page.php:12]: Removed.

Comment (nburd90) [reply home-page.php:12]: Removed.

**Pull request merged**